tooling/prepack.js

```diff
@@ -0,0 +1,5 @@
+const fs = require('fs-extra')
+
+fs.move('es', 'es_src')
```

Comment:
Proposing a more radical change: to rename current `es` to `src` (as it should be) and to build typescript straight to `/es`

Reply:
+1